### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by roscha80**
Cleaned the code
Add theme switch logic to my global color variables

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ function App() {
     history.push('usersPage')
   }
 
-  function deleteFetch(id) {
+  function withoutDeletedUserFetch(id) {
```
Comment (LisanneArdern) [inline client/src/App.js]: I would suggest naming your function with a verb and noun so it becomes clearer what the function exactly does.

File: client/src/components/Button.js [hunk 2]
```diff
@@ -14,18 +14,18 @@ export default function Button({ component: Component = 'button', ...props }) {
...
+  box-shadow: 3px 3px var(--background_contrast);
   background: ${props =>
     props.isActive ? props.theme.colors.active : props.theme.colors.background};
-  color: ${props => props.theme.colors.fontcolor};
+  color: var(--fontcolor);
   text-decoration: none;
   text-align: center;
   &:hover {
-    background: ${props => props.theme.colors.highlight};
+    background: var(--background_highlight);
```
Comment (LisanneArdern) [inline client/src/components/Button.js:28]: nice clean up!

File: client/src/components/Form.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+
+import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
 import InputField from './InputField'
 import Button from './Button'
 
 Form.propTypes = {
-  onSubmit: PropTypes.func.isRequired,
+  onSubmit: PropTypes.func,
```
Comment (LisanneArdern) [inline client/src/components/Form.js:10]: isn't a function is always required?

**Review by LisanneArdern — APPROVED** [review]
I really like this clean up! I only added a few remarks:)
Great job rob!

File: client/src/components/Form.js (hunk 3 above)
Comment (roscha80) [reply client/src/components/Form.js:10]: It isn't a must.

**Pull request merged**